lpa-submissions-e2e-tests/cypress/integration/pageFooter/pageFooter.step.js

```diff
@@ -0,0 +1,18 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { clickFooterLink } from '../../support/common/clickFooterLink';
+
+Given('LPA planning officer accesses the LPA Questionnaire', () => {
+  cy.goToTaskListPage();
+});
+
+When('they click on the {string} in the footer', (footerLink) => {
+  clickFooterLink(footerLink);
+});
+Then('the {string} correct page should be displayed', (pageUrl) => {
+  cy.verifyPage(pageUrl);
+});
+
+Then('the page title should be {string}', (pageTitle) => {
+  cy.verifyPageTitle(pageTitle);
+  cy.go('back');
```

Comment:
Why is this back link here? additional tests don't run after this so seems superfluous?

Reply:
updated as per review comments